mp/src/game/client/momentum/client_events.cpp

```diff
@@ -107,6 +107,7 @@ void CMOMClientEvents::PostInit()
 void CMOMClientEvents::LevelInitPreEntity()
 {
     //Precache();
+    PrecacheMaterial("momentum/zone_outline");
```

Comment:
Does this effectively remove the `Binding uncached material "momentum/zone_outline"` warning in console if you go on triggertests and turn around at spawn?

Also, we may want to keep this in the triggers class so that the material name can be pulled out to a macro.

If it's not precaching, you'll have to make sure `Precache()` being called inside the base trigger's `Spawn()` function.

Reply:
Yes it does remove the warning which is nice. I'll move it to the client triggers and pull it into a macro.

Reply:
Doing it this way removes the warning, but not on the first load. The `Precache()` happens before calling `DrawModel` yet it still has this error when rendering for the first time. I'm confused as to why.

For now I'll just push my changes as it's really simple to switch back if we want.

Reply:
It probably needs to be precached on the server first. I ran into this with the conc branch recently, a material would not have a model index until it was done on server.

Reply:
Wondering if its better to be in `mom_triggers` as well or just where it was before

Reply:
Can be in mom_triggers yes.